submissions/onyinyeChukwuka/css/sign-up.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
```

Comment:
This file shouldn't be here, This file should be placed in your `html-forms` folder